pyaedt/icepak.py

```diff
@@ -3182,3 +3182,446 @@ def get_face_normal(obj_face):
             self.modeler.primitives[object_name].solve_inside = False
             return boundary
         return None
+
+    @pyaedt_function_handler()
+    def assign_stationary_wall(
+        self,
+        geometry,
+        ext_condition,
+        name=None,
+        temperature="0cel",
+        heat_flux="0irrad_W_per_m2",
+        thickness="0mm",
+        htc="0w_per_m2kel",
+        htc_dataset=None,
+        ref_temperature="AmbientTemp",
+        material="Al-Extruded",  # relevant if th>0
+        radiate=False,
+        radiate_surf_mat="Steel-oxidised-surface",  # relevant if radiate = False
+        ht_correlation=False,
+        ht_correlation_type="Natural Convection",
+        ht_correlation_fluid="air",
+        ht_correlation_flow_type="Turbulent",
+        ht_correlation_flow_direction="X",
+        ht_correlation_value_type="Average Values",  # "Local Values"
+        ht_correlation_free_stream_velocity="1m_per_sec",
+        ht_correlation_surface="Vertical",  # Top, Bottom, Vertical
+        ht_correlation_amb_temperature="AmbientTemp",
+        shell_conduction=False,
+        ext_surf_rad=False,
+        ext_surf_rad_material="Stainless-steel-cleaned",
+        ext_surf_rad_ref_temp="AmbientTemp",
+        ext_surf_rad_view_factor="1",
+    ):
+        """Function to assign surface wall boundary condition.
```

Comment:
```suggestion
        """Assign surface wall boundary condition.
```
Descriptions should start with the verb describing what they do.